src/pf-notification-drawer/pf-notification-drawer.scss

```diff
@@ -0,0 +1,46 @@
+.drawer-pf {
```

Comment:
I think adding custom CSS for WCs of patternfly-webcomponents isn't unavoidable only if the structure of component is simple enough. On the other hand, Pantternfly is under no obligation to include all the css codes for derivatives like patternfly-react or patterfnly-ng. Maybe we can replace ".drawer-pf" with "pf-notification-drawer" using wc tag name to replace classname within patternfly.